packages/contracts-bedrock/src/L1/StandardValidator.sol

```diff
@@ -25,10 +25,17 @@ import { IOptimismPortal2 } from "interfaces/L1/IOptimismPortal2.sol";
 import { IPreimageOracle } from "interfaces/cannon/IPreimageOracle.sol";
 
 contract StandardValidator {
+    bytes32 constant OVERRIDES_GUARDIAN_TSTORE_SLOT = 0xea0e499a0ac72eae4682cf24b47990daf76d5744f9e4d3eb78e5846e9505073d; // keccak256("StandardValidator.overrides.guardian");
+    bytes32 constant OVERRIDES_L1PAOMULTISIG_TSTORE_SLOT =
+        0x339f30846cb2cd5e991f9a80fe2341dc2fffe74bb939549b9fbab9242e47245a; // keccak256("StandardValidator.overrides.l1PAOMultisig");
+    bytes32 constant OVERRIDES_CHALLENGER_TSTORE_SLOT =
+        0x44d7b400f8dfb79d5158f5eca2e36d9a7a2e0ca59075c45b3196ebb85445ea6e; // keccak256("StandardValidator.overrides.challenger");
```

Comment:
Ideall this would have natspec added, but I'm just noticing how painfully devoid of natspec this entire contract is, so we can defer that to a future PR that will take care of the whole file.